### PR TITLE
Localizes  image.md in Hindi

### PR DESCRIPTION
**Pull request by CloudShiva7cr**
Localizes image.md in  Hindi

### Description
Localizes content/en/docs/reference/glossary/image.md in  Hindi
 content/hi/docs/reference/glossary/image.md .

I also add text="कंटेनर in side -> {{< glossary_tooltip text="कंटेनर" term_id="container" >}} which was not in english. Because in other english docs it is present!!

### CODE REVIEW

File: content/hi/docs/reference/glossary/image.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+title: इमेज (Image)
+id: image
+date: 2018-04-12
+full_link: 
+short_description: >
+  एक कंटेनर का स्टोर्ड इंस्टेंस, जिसमें किसी एप्लिकेशन को चलाने के लिए आवश्यक सॉफ़्टवेयर का सेट हो।
```
Comment (niranjandarshann) [inline content/hi/docs/reference/glossary/image.md:7]:
```suggestion
 कंटेनर का स्टोर्ड इंस्टेंस, जिसमें किसी एप्लिकेशन को चलाने के लिए आवश्यक सॉफ़्टवेयर का सेट हो।
```

File: content/hi/docs/reference/glossary/image.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+aka: 
+tags:
+- fundamental
+---
+ एक {{< glossary_tooltip text="कंटेनर" term_id="container" >}} का स्टोर्ड इंस्टेंस, जिसमें किसी एप्लिकेशन को चलाने के लिए आवश्यक सॉफ़्टवेयर का सेट हो। 
+
+<!--more-->
+
+सॉफ़्टवेयर को पैकेज करने का एक तरीका, जो इसे कंटेनर रजिस्ट्री में स्टोर्ड करने, स्थानीय सिस्टम पर पुल करने, और एक एप्लिकेशन के रूप में चलाने की अनुमति देता है। इमेज में मेटा डेटा शामिल होता है, जो यह संकेत कर सकता है कि कौन सा एक्ज़ीक्यूटेबल चलाना है, इसे किसने बनाया, और अन्य जानकारी।
```
Comment (niranjandarshann) [inline content/hi/docs/reference/glossary/image.md:17]:
```suggestion
सॉफ़्टवेयर को पैकेज करने का एक तरीका, जो इसे कंटेनर रजिस्ट्री में स्टोर्ड करने, स्थानीय सिस्टम पर पुल (pull) करने, और एक एप्लिकेशन के रूप में चलाने की अनुमति देता है। इमेज में मेटा डेटा शामिल होता है, जो यह संकेत कर सकता है कि कौन सा एक्ज़ीक्यूटेबल चलाना है, इसे किसने बनाया, और अन्य जानकारी प्रदान कर सकता है।
```

**Review by niranjandarshann — COMMENTED** [review]
Some small suggestion  for changes

File: content/hi/docs/reference/glossary/image.md (hunk 2 above)
Comment (CloudShiva7cr) [reply content/hi/docs/reference/glossary/image.md]: @niranjandarshann Sure!! Made changes as you suggested!! Thanks!

**Review by niranjandarshann — COMMENTED** [review]
Thank you for your contribution Changes look good for me

**Review by dipesh-rawat — COMMENTED** [review]
/approve

**Pull request merged**